distribution/src/resources/config-tool/templates/conf/internal-apis.xml.j2

```diff
@@ -24,6 +24,18 @@
         <api name="{{api.api_name}}" protocol="{{api.protocols}}" class="{{api.class}}"/>
         {% endfor %}
         <api name="{{management_api.name}}" protocol="{{management_api.protocols}}" class="{{management_api.class}}">
+            {% if management_api.file_user_store.enable == true %}
```

Comment:
We need to update the documentation on this change

Reply:
Also shall we look into the possibility of moving this user store outside the management api , i.e. common to all the apis so that we can include in other handler and use for other apis as well ?

Reply:
From the way it is now, we can use the user store from all the handlers added to the management API. I prefer not adding it above this since the rest of the internal APIs are read only APIs which do not require the user store. Therefore, I think it's better to wait until we get a requiremnt?

Reply:
Recently we got a requirement to make protect Prometheus resource. I was having an idea in mind that if we move this out and make the resource path configurable for authentication, then we could use this handler for that also.

Reply:
Let's do this as a separate issue, this PR is big enough already. https://github.com/wso2/micro-integrator/issues/1450 :)